.github/workflows/pr-preview.yml

```diff
@@ -47,6 +47,7 @@ jobs:
       - name: Create Wheel File
         timeout-minutes: 120
         run: |
+          sudo apt update
```

Comment:
No clue why this is needed suddenly, but the pr-preview workflow seems to fail otherwise.